ci/jenkins/jobs/macros.yaml

```diff
@@ -196,7 +196,6 @@
 
           kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | while read IP; do
             rsync -avr --progress --inplace -e "ssh -o StrictHostKeyChecking=no -i jenkins/key/antrea-ci-key" antrea-ubuntu.tar capv@${IP}:/home/capv/antrea-ubuntu.tar
-            ssh -o StrictHostKeyChecking=no -i jenkins/key/antrea-ci-key -n capv@${IP} "sudo ctr -n=k8s.io images tag antrea/antrea-ubuntu:${DOCKER_IMG_VERSION} antrea/antrea-ubuntu"
```

Comment:
The PR says `move "ctr image tag" after "ctr image import"` but I only see it's removed, anything I missed?

I found CI was still testing master commit instead of PRs', is it related?

Reply:
Updated